code/script.js

```diff
@@ -1,51 +1,188 @@
 // Variables that point to selected DOM elements
```

Comment:
The structure of the code is very clear from top to bottom. Every section is well explained.